Makefile

```diff
@@ -1301,6 +1301,15 @@ $(GINKGO): # Build ginkgo from tools folder.
 $(GOLANGCI_LINT): # Build golangci-lint from tools folder.
 	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) $(GOLANGCI_LINT_PKG) $(GOLANGCI_LINT_BIN) $(GOLANGCI_LINT_VER)
 
+.PHONY: provider-issues
+provider-issues: # Creates issues for all CAPI providers 
+	@if [ -n "${GITHUB_ISSUE_OPENER_TOKEN}" ]; then \
```

Comment:
Can we add this check inside the go code instead of having the logic in the Makefile?

Reply:
Added via 56caf769bfb34adbd5aaec6941d05a6678cf3d14

Reply:
Updated.